examples/java/core/blob/src/main/bond/schema.bond

```diff
@@ -0,0 +1,8 @@
+﻿namespace Examples
+
+struct Example
+{
+    0: vector<blob> ListOfBlobs;
+    1: nullable<blob> NullableBlob;
+    2: blob UninitializeBlob = nothing;
```

Comment:
Point taken about keeping the IDL in the examples the same, but this really looks like a typo to me. How about fixing it in both Java and C# in this commit?